common/OpTestInstallUtil.py

```diff
@@ -280,6 +282,105 @@ def set_bootable_disk(self, disk):
         output = self.cv_SYSTEM.console.run_command(cmd)
         return
 
+    def get_boot_cfg(self):
+        """
+        Find bootloader cfg file path of host.
+
+        :return: bootloader cfg file path, empty string if no cfg file found.
+        """
+        con = self.cv_SYSTEM.cv_HOST.get_ssh_connection()
+        bootloader_cfg = [
+            '/boot/grub/grub.conf',
+            '/boot/grub2/grub.cfg',
+            '/etc/grub.conf',
+            '/etc/grub2.cfg',
+            '/boot/etc/yaboot.conf',
+            '/etc/default/grub'
+        ]
+        cfg_path = ''
+        for path in bootloader_cfg:
+            cmd = "test -f %s" % path
+            try:
+                con.run_command(cmd)
+                cfg_path = path
+            except CommandFailed:
+                continue
+        return cfg_path
+
+    def update_kernel_cmdline(self, args=None, remove_args=None, reboot=True):
+        """
+        Update default Kernel cmdline arguments
+
+        :param args: Kernel option to be included
+        :param remove_args: Kernel option to be removed
+        :param reboot: whether to reboot the host or not
+
+        :return: True on success and False on failure
+        """
+        con = self.cv_SYSTEM.cv_HOST.get_ssh_connection()
+        # check existing kernel cmdline
+        check_cmd = "cat /proc/cmdline"
+        try:
+            check_output = con.run_command(check_cmd, timeout=60)[0]
+        except CommandFailed as Err:
+            print("Failed to get currect kernel commandline using %s: %s" %
+                  (Err.command, Err.output))
+        try:
+            con.run_command("grubby --help", timeout=60)
+            cmd = 'grubby --update-kernel=`grubby --default-kernel` '
+            if args and args not in check_output:
+                cmd += '--args="%s" ' % args
```

Comment:
Does this work if the arguments are in a different order? You need to split args first and verify arg by arg i guess.